src/coreclr/inc/clrnt.h

```diff
@@ -1030,6 +1030,10 @@ RtlVirtualUnwind(
 
 #endif
 
+#ifndef IMAGE_FILE_MACHINE_LOONGARCH64
```

Comment:
Number of these `IMAGE_FILE_XXX` definitions are redundant. I am cleaning it up in #65624

Reply:
> Did you have a chance to think about this?

Yes, some coreclr-test-case runned here. (I have to retest to get which case.)

> Number of these `IMAGE_FILE_XXX` definitions are redundant. I am cleaning it up in #65624

OK, Thanks.
I will update after #65624 .

Reply:
I update the patch first and I will merge main branch after #65624 merged.